src/App.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import {
+  addTodoToServer,
+  deleteTodoFromServer,
+  editTodo,
+  getTodos,
+} from './api/api';
+import { Todo } from './types/Todo';
+
+const App = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodo, setVisibleTodo] = useState<Todo[]>([]);
+  const [searchValue, setSearchValue] = useState<Todo['title']>('');
```

Comment:
```suggestion
  const [searchValue, setSearchValue] = useState<string>('');
```
Просто string теж було б ок